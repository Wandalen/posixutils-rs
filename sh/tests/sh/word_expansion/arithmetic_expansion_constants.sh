echo $((22))
echo $((0x22ABab))
echo $((022))