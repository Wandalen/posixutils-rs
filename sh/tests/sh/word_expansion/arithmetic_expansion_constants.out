22
2272171
18
